### PR TITLE
AA-2150: Add security alerts resource (feat)

### PR DESCRIPTION
**Pull request by lukiffer**

### CODE REVIEW

File: src/security-alerts/security-alert.struct.apib [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    The description of the alert.
+
++ categories (array[string])
+    Event categories that are associated with the alert.
+
++ severity (number)
+    The severity of the alert.
+
++ status (string)
+    The status of the alert.
```
Comment (straineanu) [inline src/security-alerts/security-alert.struct.apib]:
```suggestion
    The status of the alert, any of "Open", or "Closed"
```
Matches https://github.com/armor/api-docs/blob/dev/src/logs/log-source.struct.apib#L19
QRadar has a 'hidden' too, but I doubt we'll be exposing that.

File: src/security-alerts/security-alert.struct.apib [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## SecurityAlert (object)
+
++ alertId (Guid)
+    The unique identifier of the security alert.
+
++ description (string)
```
Comment (straineanu) [inline src/security-alerts/security-alert.struct.apib]: @lukiffer In the UI story https://armor-jira.atlassian.net/browse/AA-2175
This column is named 'Summary'

File: src/security-alerts/security-alert.struct.apib (hunk 1 above)
Comment (lukiffer) [reply src/security-alerts/security-alert.struct.apib]: Updated blueprint and JSON schema.

File: src/security-alerts/security-alert.struct.apib (hunk 2 above)
Comment (lukiffer) [reply src/security-alerts/security-alert.struct.apib]: Renamed to "summary"

**Comment by lukiffer** [comment]
Nullable in TypeScript should be:

```typescript
export class Example {
    public nullableProperty?: number;
}
```

**Pull request merged**